webapp/src/pages/tagConfig/components/TagConfigComponent.js

```diff
@@ -261,12 +288,15 @@ class TagConfigComponent extends Component {
 
     tagEntryForm.push(
       <div className={"form-group row"}>
-        <button className="btn btn-primary" onClick={this.onClickAdd()}>
-          {t("Save")}
-        </button>
+        <button
+          onClick={this.onClickAdd}
+          className="btn btn-success btn-lg btn-block">
+          {t("Save Tag")}
```

Comment:
I would like this button to be on the right of the screen - what's the best way to right align?